integration-tests/opentelemetry/src/test/java/io/quarkus/it/opentelemetry/EndUserTest.java

```diff
@@ -0,0 +1,72 @@
+package io.quarkus.it.opentelemetry;
+
+import static io.restassured.RestAssured.given;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.await;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+import jakarta.inject.Inject;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.testing.exporter.InMemorySpanExporter;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import io.quarkus.it.opentelemetry.util.EndUserProfile;
+import io.quarkus.it.opentelemetry.util.EndUserResource;
+import io.quarkus.test.common.http.TestHTTPEndpoint;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import io.quarkus.test.security.TestSecurity;
+
+@QuarkusTest
+@TestHTTPEndpoint(EndUserResource.class)
+@TestProfile(EndUserProfile.class)
+public class EndUserTest {
```

Comment:
Could you duplicate this test with the default, disabled config and assert that the user data is null?
```
assertNull(attributes.get(SemanticAttributes.ENDUSER_ID));
assertNull(attributes.get(SemanticAttributes.ENDUSER_ROLE));
```

Reply:
Yes, give me a minute, I missed this comment of yours ! sorry for rerequesting the review :-)

Reply:
Done !

Reply:
No worries